src/config.json

```diff
@@ -212,7 +212,11 @@
     "cryptokitties.co",
     "remme.io",
     "jibrel.network",
-    "twinity.com"
+    "twinity.com",
+    "www.metabase.cc",
```

Comment:
we block everything under the specified domain so
`"www.metabase.network"`
is blocked by 
`"metabase.network"`